server/template.go

```diff
@@ -263,6 +263,33 @@ func init() {
 {{- end }} {{template "reviewRepoPullRequestWithTitle" .}}
 {{if .GetReview.GetBody}}>{{.Review.GetBody | replaceAllGitHubUsernames}}
 {{else}}{{end}}`))
+
+	template.Must(masterTemplate.New("helpText").Parse("" +
+		"* `/github connect{{if .EnablePrivateRepo}} [private]{{end}}` - Connect your Mattermost account to your GitHub account.\n" +
+		"{{if .EnablePrivateRepo}}" +
+		"  * `private` is optional. If used, the GitHub bot will ask for read access to your private repositories. " +
+		"If these repositories send webhook events to this Mattermost server, you will be notified of changes to those repositories.\n" +
+		"{{end}}" +
+		"* `/github disconnect` - Disconnect your Mattermost account from your GitHub account\n" +
```

Comment:
Might be an overkill, but should we extract from here the trigger name? (i.e. change all "/github" by a variable trigger name). It will probably not change, but if it were to change, we wouldn't have to update all the lines here.

Reply:
What does this add as value? Is the a use case for not using `/github` as trigger?